tests/phpunit/integration/Modules/Reader_Revenue_Manager/Web_TagTest.php

```diff
@@ -23,7 +23,7 @@ class Web_TagTest extends TestCase {
 
 	const EXPECTED_SNIPPET_STRINGS = array(
 		'Google Reader Revenue Manager snippet added by Site Kit',
-		'<script type="text/javascript" src="https://news.google.com/swg/js/v1/swg-basic.js" id="google_swgjs-js" async="async" data-wp-strategy="async"></script>', // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
```

Comment:
Here is another example (and many similar below) of a test that would have failed if we were running tests on older WP. There are a number of differences on older WP that fail here (e.g. no script strategy) and rather than using some crazy regex to account for them, I chose to simplify it to look for the script src only.